File: Python/quicksort/quicksort.py
def quicksort(arr):
    if len(arr) < 2:
        return arr
    else:
        pivot = arr[0]
        smalls = []
        bigs = []
        for i in range(1,len(arr)):
            if arr[i] < pivot:
                smalls.append(arr[i])
            else:
                bigs.append(arr[i])
        return quicksort(smalls) + [pivot] + quicksort(bigs)

print(quicksort([1,23,1,541312,4512,52,13,21,541,23,1231,412,12,32,2,32,1,2,3]))

File: JavaScript/quicksort/quicksort.js
function quicksort(arr) {
  if (arr.length < 2) {
    return arr;
  } else {
    let pivot = arr[0];
    let smalls = [];
    let bigs = [];
    for (let i = 1; i < arr.length ; i++) {
      if (arr[i] < pivot) {
        smalls.push(arr[i]);
      } else {
        bigs.push(arr[i]);
      }
    }
    return [...quicksort(smalls), pivot, ...quicksort(bigs)];
  }
}

console.log(quicksort([12,3,45,634,54,23,432,123,54,3,53,1,52,23,12,3542,43]))
